app/kuma-dp/pkg/config/validate.go

```diff
@@ -26,10 +29,36 @@ func ValidateTokenPath(path string) error {
 		return errors.Wrapf(err, "could not read the token in the file %s", path)
 	}
 
-	_, _, err = new(jwt.Parser).ParseUnverified(string(rawToken), &jwt.MapClaims{})
+	token, parts, err := new(jwt.Parser).ParseUnverified(string(rawToken), &jwt.MapClaims{})
 	if err != nil {
-		return errors.Wrapf(err, "token in the file %s is not valid JWT token. Double check blank characters in the file", path)
+		return errors.Wrapf(err, "token in the file %s is not valid JWT token. Double check for blank characters in the file", path)
+	}
+
+	core.Log.Info("dump ", "token", token, "parts ", parts)
```

Comment:
`dump`?
Also - I don't think we should print the token in the logs for security reasons.

Reply:
Oh, my debug logs leaked into the PR ...